content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+
+![Access API process](/img/user-guide-for-authorization.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, the user must log into the [HPE GreenLake Console,](https://common.cloud.hpe.com) have deployed the DSCC to the intended region, and onboarded a storage array (HPE Alletra, HPE Nimble, or HPE Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. For instance, the user must have volume management capability in the Data Ops Management to create a storage volume in US region. For more information about the role based access control, please take a look at the [HPE GreenLake User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=ccs-help_en_us)
+
+The API Gateway menu is available inside the Manage menu. From HPE Greenlake Console click on Menu to access this Manage Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway menu provides the following operations:
+
+1. Create and manage API client credential association in order to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals).
+   2. Client ID (Hexadecimals).
+   3. Client Secret (Hexadecimals).
+   4. URL to the HPE GreenLake Console API end-point (string).
+   5. URL to the HPE DSCC API end-point (string).
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each instance of the API client credential represents the authorization relationship between the client application and the DSCC REST API end point. Click on the Create Credentials button to generate a client credential. Afterward the user can obtain the client secret and client secret, and used them to generate the access token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. For the purpose of using the DSCC REST API, please select Data Services Cloud Console instance in the correct region where the array has been deployed. The list shows all instances with the region where the applications are deployed.
+
+![select the application](/img/select-the-desired-application.png "Choose application (DSCC)")
+
+After selecting the correct application, enter the Credential Name (Please see HPE GreenLake Cloud Console User Guide for supported characters).  Click the Create Credentials button to proceed with Client Credentials creation.
```

Comment:
I think I forgot to mention this one earlier... don't you want to link to the user guide here when you say HPE GreenLake Cloud Console User Guide?